### PR TITLE
Improve Python 2 first tutorial

### PR DESCRIPTION
**Pull request by alicestrt**
I have just started learning Python with the help of @lwm and Codebar and finished all three tutorials. Having had my first experience with a Python tutorial, I found some additional explanation very helpful, and thus added it to the given tutorial, in the hope it will be useful for other beginners.

This is just the first commit. If you think it's going in the right direction, I can happily add more.

### CODE REVIEW

File: python/lesson1/tutorial.md [hunk 1]
```diff
@@ -20,17 +20,34 @@ On Windows:
...
 
-    Python 2.7.6 (default, Sep  9 2014, 15:04:36)
+On Mac OS / GNU/Linux, open the Terminal and type (without the `$` sign)
 
-If you see it, make sure you're running Python 2, not Python 3, otherwise parts of this exercise will not make sense. Once you've checked that, we're ready to move on to writing Python.
+    $ python --version
+
+What you should see is:
+
+    python 2.7.12
```
Comment (lwm) [inline python/lesson1/tutorial.md]: Oh, forgot the `$` here at the start of the command.

File: python/lesson1/tutorial.md [hunk 2]
```diff
@@ -20,17 +20,34 @@ On Windows:
...
+
+Any 2.* version should do for this tutorial to work.
+
+Now, open the *REPL* by typing:
+
+    python
```
Comment (lwm) [inline python/lesson1/tutorial.md]: Oh, also here. We forgot the `$` as well.

File: python/lesson1/tutorial.md [hunk 3]
```diff
@@ -20,17 +20,34 @@ On Windows:
...
+
+And you should see something like this:
+
+Python 2.7.12 (default, Jul  1 2016, 15:12:24)
```
Comment (lwm) [inline python/lesson1/tutorial.md]: Ah, this block of code needs to be indented by four spaces, so it will get the correct formatting.

**Comment by KimberleyCook** [comment]
@alicestrt would you like me to merge this one then do another PR or let you continue making amends to this PR?

**Comment by alicestrt** [comment]
@KimberleyCook I will address the issues @lwm mentioned and then notify you when it's ready to be merged. Thanks!!!

File: python/lesson1/tutorial.md [hunk 4]
```diff
@@ -20,17 +20,34 @@ On Windows:
...
+On Windows, open `cmd` program and type (without the `$` sign):
 
-On Mac OS, open the Terminal and type "python" to start the REPL. At the top, you'll see something similar (but probably not identical) to this:
+    $ python --version
 
-    Python 2.7.6 (default, Sep  9 2014, 15:04:36)
+On Mac OS / GNU/Linux, open the Terminal and type (without the `$` sign)
 
-If you see it, make sure you're running Python 2, not Python 3, otherwise parts of this exercise will not make sense. Once you've checked that, we're ready to move on to writing Python.
+    $ python --version
```
Comment (KimberleyCook) [inline python/lesson1/tutorial.md]: As this command is the same regardless of what machine you're using do you think they need splitting up. Maybe combine lines 27 and 31?

Comment (alicestrt) [inline python/lesson1/tutorial.md]: Yes, I did that.

Comment (KimberleyCook) [inline python/lesson1/tutorial.md]: Nice, let me know when you want me to merge.

**Comment by lwm** [comment]
LGTM!

**Comment by alicestrt** [comment]
@KimberleyCook if you think it's all good, it's ready on my side.

**Comment by KimberleyCook** [comment]
All merged, huge thanks to you both

**Pull request merged**